src/convertToObject.js

```diff
@@ -5,8 +5,28 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const getDeclarationAsStrings = sourceString
+    .split(';')
+    .map((rawDeclaration) => rawDeclaration.trim());
+  const getArrayOfDeclarationEntries = getDeclarationAsStrings.map(
+    (preparedDeclaration) =>
+      preparedDeclaration
+        .split(':')
+        .map((preparedDeclarationToTrim) => preparedDeclarationToTrim.trim()),
+  );
+
+  return getArrayOfDeclarationEntries.reduce((acc, declarationEntry) => {
+    if (declarationEntry.length === 1) {
+      return acc;
+    }
+
+    return {
+      ...acc,
+      [declarationEntry[0]]: declarationEntry[1],
```

Comment:
The code does not handle the case where a CSS property might have an important flag (e.g., `!important`). This will be included in the value, which might not be the desired behavior. Consider adding logic to handle this case appropriately.